src/Components/MainSection/TaskList/TaskList.css

```diff
@@ -0,0 +1,6 @@
+.main-section-container {
+	padding: 2rem 1rem;
+	margin: 1.5rem;
+	border-radius: 1rem;
+	background-color: lightpink;
```

Comment:
you can use variables for color naming

Reply:
These are just for visualising the layout, I'll add variables later